packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/styles.ts

```diff
@@ -123,17 +123,11 @@ export function getStylesConfig(wco: WebpackConfigOptions) {
   }
 
   let sassImplementation: {} | undefined;
-  let fiber: {} | undefined;
```

Comment:
Why were the `fiber` parts removed?

Reply:
Sass-loader handles it internally now.
Ref: https://github.com/webpack-contrib/sass-loader/blob/master/CHANGELOG.md#features